src/java/picard/analysis/directed/HsMetrics.java

```diff
@@ -55,76 +55,76 @@
     public long PF_UNIQUE_READS;
 
-    /** PF reads / total reads.  The percent of reads passing filter. */
+    /** The fraction of reads passing the vendor's filter, PF_READS/TOTAL_READS.   */
     public double PCT_PF_READS;
 
-    /** PF Unique Reads / Total Reads. */
+    /** The fraction of PF_UNIQUE_READS from the TOTAL_READS, PF_UNIQUE_READS/TOTAL_READS. */
     public double PCT_PF_UQ_READS;
 
-    /** The number of PF unique reads that are aligned with mapping score > 0 to the reference genome. */
+    /** The number of PF_UNIQUE_READS that aligned to the reference genome with a mapping score > 0. */
     public long PF_UQ_READS_ALIGNED;
 
-    /** PF Reads Aligned / PF Reads. */
+    /** The fraction of PF_UQ_READS_ALIGNED from the total number of PF reads. */
     public double PCT_PF_UQ_READS_ALIGNED;
 
-    /** The number of PF unique bases that are aligned with mapping score > 0 to the reference genome. */
+    /** The number of PF unique bases that are aligned to the reference genome with mapping scores > 0. */
     public long PF_BASES_ALIGNED;
 
-    /** The number of bases in the PF aligned reads that are mapped to a reference base. Accounts for clipping and gaps. */
+    /** The number of bases in the PF_UQ_READS_ALIGNED reads. Accounts for clipping and gaps. */
     public long PF_UQ_BASES_ALIGNED;
 
-    /** The number of PF aligned bases that mapped to a baited region of the genome. */
+    /** The number of PF_BASES_ALIGNED that are mapped to the baited regions of the genome. */
     public long ON_BAIT_BASES;
 
-    /** The number of PF aligned bases that mapped to within a fixed interval of a baited region, but not on a baited region. */
+    /** The number of PF_BASES_ALIGNED that are mapped to within a fixed interval containing a baited region, but not within the baited section per se. */
     public long NEAR_BAIT_BASES;
 
-    /** The number of PF aligned bases that mapped to neither on or near a bait. */
+    /** The number of PF_BASES_ALIGNED that are mapped distal to a baited region. */
     public long OFF_BAIT_BASES;
 
-    /** The number of PF aligned bases that mapped to a targeted region of the genome. */
+    /** The number of PF_BASES_ALIGNED that are mapped to a targeted region of the genome. */
     public long ON_TARGET_BASES;
 
-    /** On+Near Bait Bases / PF Bases Aligned. */
+    /** The fraction of PF_BASES_ALIGNED located on or near a baited region (ON_BAIT_BASES + NEAR_BAIT_BASES)/PF_BASES_ALIGNED. */
     public double PCT_SELECTED_BASES;
 
-    /** The percentage of aligned PF bases that mapped neither on or near a bait. */
+    /** The fraction of PF_BASES_ALIGNED that are mapped distal to a bait, OFF_BAIT_BASES/PF_BASES_ALIGNED.  */
     public double PCT_OFF_BAIT;
 
-    /** The percentage of on+near bait bases that are on as opposed to near. */
+    /** The fraction of bases on or near baits that are within a baited region, ON_BAIT_BASES/(ON_BAIT_BASES + NEAR_BAIT_BASES). */
```

Comment:
"that are within a baited region" still feels like it could include both "on" and "near" cases, because "region" can be interpreted as the overall targeted region, rather than the exact positions covered by baits. Should this perhaps be more specific and say something like "at positions covered by baits"?